src/pages/home/report/ReportActionsView.js

```diff
@@ -397,6 +425,68 @@ class ReportActionsView extends React.Component {
         }
     }
 
+    /**
+     * Scrolls to a specific report action ID
+     */
+    scrollToReportActionID() {
+        this.actionIndexID = _.findIndex(this.sortedReportActions, (
+            ({action}) => parseInt(action.reportActionID, 10) === this.props.reportActionID
+        ));
+
+        if (this.actionIndexID !== -1) {
+            ReportScrollManager.scrollToIndex({index: this.actionIndexID, viewPosition: 0.5});
+            this.setState({shouldHighlightReportActionID: true});
+        }
+    }
+
+    /**
+     * Records when a report actionID is done rendering.
+     *
+     * @param {string} reportActionID
+     */
+    recordReportActionIDRendered(reportActionID) {
+        this.renderedActionIDs.add(parseInt(reportActionID, 10));
+        this.checkScrollToReportAction();
+    }
+
+    /**
+     * Records when our FlatList is done measuring the heights and offset of items. Since native will hit this block immediately we'll use a param checkScrollToReportAction to prevent
+     * calling the scroll method unnecessarily until the item has been rendered.
```

Comment:
Correct yes, native hits this block immediately.

> Do we still need to have the "doneMeasuring = true" when we are on native? Seems like it would be true that if we are on native then we don't need to measure anything?

In my current logic yes because ReportActionsView does not know if its native or not so i'm tracking if doneMeasuring is true and item is done rendering. We still want to set the flag doneMeasuring as true here immediately for native so that when the item is rendered and we call `checkScrollToReportAction` it knows measuring is done and item is rendered so its good to scroll.